hamp_radar/iquick.py

```diff
@@ -24,6 +55,25 @@ class SingleMainBlockGeometry:
 
 @dataclass
 class MultiMainBlockGeometry:
+    """
+    Representation of multiple main blocks as list of SingleSubBlockGeometry
+    instances with some overarching metadata. This can be multiple 'main chunks'
+    as in Meteorological Ka-Band Cloud Radar MIRA35 Manual, section 2.3.2
+    'File structure'.
+    
+    Allows for 'count' number of SingleMainBlockGeometry instances to be
+    combined, assuming they are similar enough (e.g. same subblocks, same offset,
+    same tag etc.). (See compact_geometry function.)
+
+    Attributes:
+        tag (str): The tag/signature describing the main blocks.
+        offset (int): Pointer to the start of the first main block.
+        count (int): The number of SingleMainBlockGeometry instances
+                     combined to make the MultiMainBlockGeometry instance.
```

Comment:
It's not a combination of sub blocks, it's a combination of main blocks.
```suggestion
        count (int): The number of main blocks combined.
```